Frontend/CO.CDP.OrganisationApp/Pages/Registration/JoinOrganisation.cshtml

```diff
@@ -36,23 +37,23 @@
                 <dl class="govuk-summary-list">
```

Comment:
Missed "User details" heading just above this line

Reply:
Done